src/components/Modals/Send/views/Details.tsx

```diff
@@ -136,7 +141,11 @@ export const Details = () => {
                   {cryptoSymbol}
                 </Button>
               }
-              inputRightElement={<SendMaxButton onClick={handleSendMax} />}
+              inputRightElement={
+                !(wallet?.getVendor() === 'WalletConnect') ? (
```

Comment:
Prefer `x !== y` vs. `!(x === z)`, no reason not to go for the arithmetical not equal unless you're doing multiple not equal checks and want to use only one `!` for cleanness